### PR TITLE
Cuda manual seed all

### PR DESCRIPTION
**Pull request by LokeZhou**
### PR Docs
https://github.com/PaddlePaddle/docs/pull/6100
### PR APIs
```bash
 torch.cuda.manual_seed_all
torch.cuda.initial_seed()
```

### CODE REVIEW

File: tests/test_cuda_manual_seed_all.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    pytorch_code = textwrap.dedent(
+        """
+        import torch
+        result = None
+        if torch.cuda.is_available():
+            torch.cuda.manual_seed_all(123)
+            result = torch.initial_seed()
+        """
+    )
+    obj.run(pytorch_code, ["result"], check_value=False)
```
Comment (zhwesky2010) [inline tests/test_cuda_manual_seed_all.py:32]: 这个check_value应该设置为True，两者的seed都为123

Comment (LokeZhou) [reply tests/test_cuda_manual_seed_all.py:32]: 设置了seed依旧无法对齐

Comment (LokeZhou) [reply tests/test_cuda_manual_seed_all.py]: done,新增torch.cuda.initial_seed() 转换

**Review by zhwesky2010 — APPROVED** [review]
LGTM，顺便补一下torch.cuda.initial_seed的文档吧

**Comment by LokeZhou** [comment]
> LGTM，顺便补一下torch.cuda.initial_seed的文档吧

done  https://github.com/PaddlePaddle/docs/pull/6111
